ch1.md

```diff
@@ -67,32 +75,33 @@ Well, this time we got the right answer. With much less code. The function nesti
 There's really nothing special at all about these two functions other than their names. Let's rename our custom functions to `multiply` and `add` in order to reveal their true identities.
 
 ```js
-var add = function(x, y) { return x + y; };
-var multiply = function(x, y) { return x * y; };
-
-var flock_a = 4;
-var flock_b = 2;
-var flock_c = 0;
-
-var result = add(
-  multiply(flock_b, add(flock_a, flock_c)), multiply(flock_a, flock_b)
-);
+// A Running Example: https://jsfiddle.net/7g2brqaf/
+const
+  add      = (x, y) => x + y,
+  multiply = (x, y) => x * y;
+
+const
+  flock_a = 4,
+  flock_b = 2,
+  flock_c = 0,
+  result =
+    add(multiply(flock_b, add(flock_a, flock_c)), multiply(flock_a, flock_b));
 //=>16
 ```
 And with that, we gain the knowledge of the ancients:
 
 ```js
 // associative
-add(add(x, y), z) === add(x, add(y, z));
+add(add(x, y), z) == add(x, add(y, z));
 
 // commutative
-add(x, y) === add(y, x);
+add(x, y) == add(y, x);
 
 // identity
-add(x, 0) === x;
+add(x, 0) == x;
 
 // distributive
-multiply(x, add(y,z)) === add(multiply(x, y), multiply(x, z));
+multiply(x, add(y,z)) == add(multiply(x, y), multiply(x, z));
```

Comment:
TBH I think I was contemplating using just `=` since these should be mathematical equations and sine its wasn't runnable code but decided against it and only added 1 additional `=`.

I have no strong preference either way.